client/dashboard/sites/overview/plan-card.tsx

```diff
@@ -0,0 +1,43 @@
+import { useQuery } from '@tanstack/react-query';
+import { cloneElement } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+import { wordpress } from '@wordpress/icons';
+import { siteCurrentPlanQuery } from '../../app/queries/site-plans';
+import { sitePurchaseQuery } from '../../app/queries/site-purchases';
+import OverviewCard from '../overview-card';
+import type { Site, Plan, Purchase } from '../../data/types';
+
+export default function PlanCard( { site }: { site: Site } ) {
+	const { data: plan, isLoading: isLoadingPlan } = useQuery( siteCurrentPlanQuery( site.ID ) );
+	const { data: purchase, isLoading: isLoadingPurchase } = useQuery( {
+		...sitePurchaseQuery( site.ID, plan?.id ?? '' ),
+		enabled: !! plan?.id,
```

Comment:
Curiously, `plan.id` is not the ID of a plan, but appears to be the ID of a purchase. So it is `null` when it is a free plan.